.travis.yml

```diff
@@ -4,6 +4,14 @@ node_js:
 - '4'
 - '6'
 
+# Run on travis with both yarn enabled and disabled.
+env:
+- YARNPKG=true
+- YARNPKG=false
+before_install:
+- if [[ $YARNPKG != "true" ]]; then rm yarn.lock; fi
```

Comment:
This is turned into a matrix 2x2, and so we will have 4 builds:

- nodejs 4 without yarn
- nodejs 4 with yarn
- etc.

like in the build associated to this PR: https://travis-ci.org/mozilla/web-ext/builds/188911714

and so we don't need the `else` branch, because we only remove the lock file when yarn is not enabled, so that travis does not "auto-install and enable yarn" as the needed installer and proceed to use the regular npm command as usually